src/redturtle/prenotazioni/behaviors/booking_folder/__init__.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+import os
+
+from redturtle.prenotazioni.io_tools.api import Api
+
+
+class BookingNotificationSupervisorUtility:
+    """Supervisor to allow/deny the specific
+    notification type according to business logic"""
+
+    def is_email_message_allowed(self, booking):
+        if getattr(
+            booking.getPrenotazioniFolder(), "notifications_email_enabled", False
+        ):
+            return True
+
```

Comment:
i would move the check of "email" compiled field here, because you need to use this method also on sms one